src/freeze-dry/inline-styles.test.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-env jest */
+
+import inlineStyles from 'src/freeze-dry/inline-styles'
+import * as common from 'src/freeze-dry/common'
+
+fetch.mockResponse(new Blob(['Dummy Styles']), { 'status': 200, 'statusText': 'Response Accepted' })
+
+describe('inlineStyles', () => {
+    test('should return create a <style> with the fetch of stylesheet', async () => {
+        const parser = new DOMParser()
+        const rootElement = parser.parseFromString(
+            '<html><head><link rel="stylesheet" type="text/css" href="https://example.com/theme.css"></head></html>',
+            'text/html'
+        )
+        const docUrl = 'https://example.com'
+        await inlineStyles({rootElement, docUrl})
+        expect(rootElement.querySelector('style').innerHTML).toBe('Dummy Styles')
+    })
+
+    test('should convert the url in <style> to dataUris', async () => {
+        expect.assertions(2)
+        const dataUri = 'data:text/plain;charset=utf-8;base64,aHR0cHM6Ly9leGFtcGxlLmNvbS9wdWJsaWMvaW1hZ2UvYmFja2dyb3VuZC5qcGVn'
+        const spy = jest.spyOn(common, 'urlToDataUri').mockReturnValue(dataUri)
+        const parser = new DOMParser()
+        const rootElement = parser.parseFromString(
+            '<html><head><style type="text/css">div{background-image: url("public/image/background.jpeg");}</style></head><div></div></html>',
+            'text/html'
+        )
+        const docUrl = 'https://example.com'
+        await inlineStyles({rootElement, docUrl})
+        expect(spy).toHaveBeenCalled()
+        expect(rootElement.querySelector('style').innerHTML).toBe(`div{background-image: url(${dataUri});}`)
+        spy.mockRestore()
+    })
```

Comment:
Great. I would perhaps just extract the long strings, and make it obvious to the reader that the data URL matches the stylesheet contents; e.g.:
```
const styleSheet = 'div{background-image: url("public/image/background.jpeg");}'
const styleSheetAsDataUri = `data:text/plain;charset=utf-8;base64,aHR0cHM6Ly9leGFtcGxlLmNvbS9wdWJsaWMvaW1hZ2UvYmFja2dyb3VuZC5qcGVn`
```
(or use [`btoa`](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/btoa) to convert the string on the spot, but note it has trouble with unicode, string must remain ascii)